### ISSUE TITLE
Can't install!

### ISSUE THREAD

**Issue by ttwajsb** [itsMeDavidV/Watchla-Support#135]
Just downloaded the latest version 3.4.3 (288)

Watcha no longer showed All Commands list so I tried to uninstall and reinstall it on my Watch. See 8-sec video of what happens now. I'm completely stuck. Can't use Watchla. :( 

https://share.icloud.com/photos/073jLD86UuaPcvuQOOvdet0mA

**Comment by ttwajsb** [comment]
Just downloaded the latest version 3.4.3 (288)

Watcha no longer showed All Commands list so I tried to uninstall and reinstall it on my Watch. See 8-sec video of what happens now. I'm completely stuck. Can't use Watchla. :( 

https://share.icloud.com/photos/073jLD86UuaPcvuQOOvdet0mA

**Comment by ttwajsb** [comment]
URGENT Leaving on a driving trip early Sun morning and still getting same result, even after removing app from Watch and IPhone and reinstalling from scratch from the App Store. Please help me not have to keyboard my PIN ask week. Thanks!

**Comment by itsMeDavidV** [comment]
Hi @ttwajsb unfortunately Watchla has no control over downloading or installing apps. This is an issue for Apple Support :/

But also keep in mind you can bypass the phone completely and download Watchla to your Apple Watch directly from the App Store app on your Apple Watch itself.

**Comment by ttwajsb** [comment]
After uninstalling the app, restarting both devices, then reinstalling both the iPhone app and then on the Watch the Watch app, I finally got it to work. So glad to have it working again, though I had to give up on the Bluetooth feature after trying really hard to set it up. I'm on a Series 4 Watch; maybe that's the problem.


I wonder if somewhere in the Help docs you might want to alert people that in rare cases this may occur? (If it says so already I obvs missed it :/ )
